usagov/usagov-2021
# dev: USAGOV-2026 - Breadcrumbs overlap other text on scams tool

Fixing a theming issue where certain pages had their breadcrumb overlap with the content underneith it.


https://cm-jira.usa.gov/browse/USAGOV-2026

## Description
Changed the CSS that force-set the height for breadcrumbs [that was previously too short for certain pages], to be a min-height instead of a static-height. This allows the breadcrumb contain to stretch taller as needed.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
After merging in, or switching branches. Go to the problem-oage of https://localhost/where-report-scams/report-unemployment-identity-theft#block-usagov-content or any page that has a long-long breadcrumb. See the Jira ticket for the screenshot I have in my comment of my local (before fixing the problem).
We should also test other pages to make sure no [non-problem] breadcrumbs got changed inadvertently.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
